<launch>
        <!-- Set parameters for the node -->
        <param name="camera_name" value="sony_cam2"/>
        <param name="dictionary" value="DICT_7X7_1000"/>
        <param name="fiducial_len" value="0.020"/>
        <param name="visualize" value="true"/>
        <param name="corner_refine_criteria" value="[3,20,0.001]"/>

        <!-- Node definition -->
        <node name="aruco_detector" pkg="tesol_detect" type="aruco_det.py" output="screen">
        </node>
</launch>
